multiscale_benchmark/2022_09_hackathon/Biodynamo/use_case_2Ddisk/code_scripts/src/custom_ops.h

```diff
@@ -19,6 +20,21 @@ struct MoveCellsPlane : public AgentOperationImpl {
 
 };
 
+  struct CountCells : public StandaloneOperationImpl {
+    // This macro will generate the boilerplate code. It must be included.
+    BDM_OP_HEADER(CountCells);
+
+    void operator()() override {
+      auto* sim = Simulation::GetActive();
+      auto* rm = sim->GetResourceManager();
+
+      (*total_cells_).push_back(rm->GetNumAgents());
+    }
+
+    std::vector<size_t>* total_cells_;
```

Comment:
@nicogno I think I'd go for a vector rather than a vector to a pointer. I'm not sure if the vector is ever initialized 🤔 
```
    void operator()() override {
       auto* sim = Simulation::GetActive();
       auto* rm = sim->GetResourceManager();

       total_cells.push_back(rm->GetNumAgents());
     }

     std::vector<size_t> total_cells;
```

Reply:
Good idea, fixed now